infinite/index.ts

```diff
@@ -267,4 +267,10 @@ export const infinite = (<Data, Error>(useSWRNext: SWRHook) =>
   }) as unknown as Middleware
 
 export default withMiddleware(useSWR, infinite) as SWRInfiniteHook
-export { SWRInfiniteConfiguration, SWRInfiniteResponse, InfiniteFetcher }
+export {
+  SWRInfiniteConfiguration,
+  SWRInfiniteResponse,
+  SWRInfiniteHook,
+  InfiniteKeyLoader,
```

Comment:
Maybe we should rename to `SWRInfiniteKeyLoader` and `SWRInfiniteFetcher `?

Reply:
thanks! updated in aeaeeaa , this could break the code below, not sure if it's a concern tho.

```ts
import { InfiniteKeyLoader } from 'swr/infinite/dist/infinite/types';
```

Reply:
Thanks, we don't guarantee the subpath imports like this work as expected since it could change during iterations. Allowed subpaths exports are marked in package.json exports field.

Reply:
gotcha, thx!